independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/BeanManagerImpl.java

```diff
@@ -115,7 +115,7 @@ public void validate(InjectionPoint injectionPoint) {
         throw new UnsupportedOperationException();
     }
 
-    @Override
+    // Deprecated method which can be safely removed once we use CDI 4.0+
```

Comment:
I'd actually love to perform some logging here stating you should use a different approach but I am sort of expecting complaints from users seeing an info/warn messages all of sudden so I didn't do it yet.